packages/security/src/deviceTrust/checks/DebuggerCheck.ts

```diff
@@ -0,0 +1,22 @@
+import { SecurityCheck } from "../SecurityCheck";
+import { SecurityCheckResult } from "../SecurityCheckResult";
+
+declare let cordova: any;
+
+export class DebuggerCheck implements SecurityCheck {
+    public readonly name = "Is Debugger Check";
+
+    public check(): Promise<SecurityCheckResult> {
+        return new Promise((resolve, reject) => {
+            if (!cordova.plugins.isDebug) {
```

Comment:
`cordova.plugin` would be `undefined` if not in Cordova (aka browser) should we consider this @wtrocki ? I think it's the same in the other Security check.

Reply:
It's unfortunate because plugin itself is using ancient namespace: https://github.com/mattlewis92/cordova-plugin-is-debug/blob/master/plugin.xml#L8
We need to check all starting from `window.cordova`